exec/jdbc-all/pom.xml

```diff
@@ -575,7 +575,7 @@
 
         <build>
           <plugins>
-            <plugin>
+            <plugin> <!-- TODO: this plugin has common things with default profile. Factor out this common things to avoid duplicate code -->
```

Comment:
Could you please implement this TODO? It doesn't look to be complicated.

Reply:
ok :)

Reply:
Ok, thanks!